### DISCUSSION TITLE
Support for 'git bundle' command

### DISCUSSION THREAD

**Discussion by Nightreaver** [gitpython-developers/GitPython#1348] [Q&A] [answered]
Hello

I've been checking the docs, the git repo and google, but i cannot find any connection from gitpython to the 'git bundle' command. Tho i need it pretty quickly, i wanted to ask if this can be (or is already?) implemented.

I'm talking about this

https://git-scm.com/docs/git-bundle

**Comment by Byron** [comment, marked as answer]
I think what's closest to solving your issue is [described here: using git directly](https://gitpython.readthedocs.io/en/stable/tutorial.html#using-git-directly). There is no specific implementation of it.
